### ISSUE TITLE
Target "_ResolveCssScopes" does not exist (dotnet 8.0.0-rc2)

### ISSUE THREAD

**Issue by laurence-myers** [fsbolero/Template#42]
## Error

I created a new project from the template and tried to run the server. It gave me this error:

```
Bolero.Build.targets(21,41): Error MSB4057 : The target "_ResolveCssScopes" does not exist in the project.
```

## Setup

```
dotnet new -i Bolero.Templates

Welcome to .NET 8.0!
---------------------
SDK Version: 8.0.100-rc.2.23502.2
...snip...
Success: Bolero.Templates::0.22.7 installed the following templates:
Template Name              Short Name  Language  Tags
-------------------------  ----------  --------  -----------------
Bolero Hosted Application  bolero-app  F#        Web/Blazor/Bolero
```

## Create project

```
dotnet new bolero-app -o Baffo
```

## Run project

```
PS C:\Users\...\code\baffo-fs> dotnet run --project ./src/Baffo.Server
C:\Users\...\.nuget\packages\bolero.build\0.22.45\build\Bolero.Build.targets(21,41): error MSB4057: The target "_ResolveCssScopes" does not exist 
```

## Build targets file

```
<?xml version="1.0" encoding="utf-8"?>
<Project>
  <PropertyGroup Condition="'$(BoleroTaskAssemblyPath)' == ''">
    <BoleroTaskAssemblyPath>$(MSBuildThisFileDirectory)..\tools\Bolero.Build.dll</BoleroTaskAssemblyPath>
    <ProduceReferenceAssembly>false</ProduceReferenceAssembly><!-- See https://github.com/fsbolero/Bolero/issues/279 -->
  </PropertyGroup>
  <PropertyGroup>
    <_BoleroScopedCssSourceFile>$(BaseIntermediateOutputPath)CssScopes.fs</_BoleroScopedCssSourceFile>
  </PropertyGroup>

  <UsingTask AssemblyFile="$(BoleroTaskAssemblyPath)" TaskName="Bolero.Build.BoleroStripFSharpMetadata"
             Condition="'$(BoleroStripAssemblies)' != 'False'" />
  <UsingTask AssemblyFile="$(BoleroTaskAssemblyPath)" TaskName="Bolero.Build.BoleroApplyCssScopes" />

  <!-- Strip F# metadata embedded files -->
  <Target Name="_BoleroStripFSharpMetadata" AfterTargets="ILLink" Condition="'$(BoleroStripAssemblies)' != 'False'">
    <BoleroStripFSharpMetadata AssembliesDir="$(IntermediateLinkDir)" ReferencePath="@(ManagedAssemblyToLink)" />
  </Target>

  <!-- Generate the CSS Scope identifier for *.bolero.css -->
  <Target Name="_BoleroComputeCssScope" DependsOnTargets="_ResolveCssScopes">
    <ComputeCssScope ScopedCssInput="@(BoleroScopedCss)" Targetname="$(TargetName)">
      <Output TaskParameter="ScopedCss" ItemName="_BoleroScopedCss" />
    </ComputeCssScope>
  </Target>

  <!-- Generate CssScopes.fs and add *.bolero.css to Razor's _ScopedCss -->
  <Target Name="_BoleroApplyCssScopes" DependsOnTargets="_BoleroComputeCssScope" BeforeTargets="_ResolveScopedCssOutputs;CoreCompile"
          Inputs="@(_BoleroScopedCss);$(MSBuildThisFileFullPath)" Outputs="$(_BoleroScopedCssSourceFile)">
    <BoleroApplyCssScopes ScopedCss="@(_BoleroScopedCss)" ScopedCssSourceFile="$(_BoleroScopedCssSourceFile)" />
    <ItemGroup>
      <CompileBefore Include="$(_BoleroScopedCssSourceFile)" />
      <_ScopedCss Include="@(_BoleroScopedCss)" />
    </ItemGroup>
  </Target>
</Project>
```

The offending line is `<Target Name="_BoleroComputeCssScope" DependsOnTargets="_ResolveCssScopes">`. I see no other reference to `_ResolveCssScopes`.

**Comment by laurence-myers** [comment]
Ah, I see this is a Bolero issue.

https://github.com/fsbolero/Bolero/issues/317

I hope once it's fixed in Bolero, some kind soul will update the template too. 🙏😊

**Comment by Tarmil** [comment]
This has now been released in v0.23.
